templates/all-services/index.json-ld

```diff
@@ -102,6 +102,13 @@
 		}
 	],
 	"iteration": [
+		{
+			"@id": "_:iteration_allservices_2",
+			"name": "All services page - Iteration 2",
+			"date": "2025-01",
+			"additions": [ "Adding expand/collapse all buttons." ],
+			"detectableBy": "Three navigation bands followed by a series of <details> elements for all goc themes and topics."
```

Comment:
Add a mention about the expand/collapse button which is what makes it different from iteration 1.